src/content/docs/zh-cn/guides/deploy/cloudflare.mdx

```diff
@@ -7,7 +7,11 @@ i18nReady: true
 
 你可以将你的 Astro 项目部署在 [Cloudflare Pages](https://pages.cloudflare.com/) 上。 Cloudflare Pages 是一个供前端开发人员协作和部署静态 (JAMstack) 或 SSR 网站的平台。
 
-本指南包含有关通过 Pages Dashboard 或使用 Wrangler Cloudflare CLI (命令行) 来部署到 Cloudflare Pages 的相关说明。
+本指南包含：
+
+- [如何使用-git-部署一个站点](#如何使用-git-部署一个站点)
```

Comment:
Maybe these three need a better translation, I think...

Reply:
Didn't notice, it has been modified. I used the title of the original translation directly here.